src/Symfony/Bridge/Twig/Tests/Extension/FormExtensionTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Tests\Extension;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Twig\Extension\FormExtension;
+use Symfony\Bridge\Twig\Form\TwigRendererInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Form\FormRendererInterface;
+use Twig\Environment;
+
+class FormExtensionTest extends TestCase
```

Comment:
should I mark this test somehow as deprecated or similar? There is no need to merge it into master for example as there is nothing to test anymore really